hpOneView/oneview_client.py

```diff
@@ -513,6 +515,18 @@ def power_devices(self):
         return self.__power_devices
 
     @property
+    def unmanaged_devices(self):
+        """
+        Gets the Unmanaged Devices API client.
+
+        Returns:
+            Unmanaged Devices:
```

Comment:
We don't have a type "Unmanaged Devices", actually  the returned type is 'dict'.